integration-cli/docker_cli_import_test.go

```diff
@@ -22,8 +22,15 @@ func TestImportDisplay(t *testing.T) {
 		t.Errorf("import failed with errors: %v, output: %q", err, out)
 	}
 
-	if n := strings.Count(out, "\n"); n != 2 {
-		t.Fatalf("display is messed up: %d '\\n' instead of 2", n)
+	// TODO: remove check after drop of go1.2 support
+	var expected int
+	if strings.HasPrefix(goVersion, "go1.2") {
+		expected = 2
+	} else {
+		expected = 3
+	}
+	if n := strings.Count(out, "\n"); n != expected {
+		t.Fatalf("display is messed up: %d '\\n' instead of %d, go version: %s", n, expected, goVersion)
```

Comment:
I'm not so sure this is correct -- why does Go 1.2 generate three lines instead of two?  What are they?